### PR TITLE
reorg(integration)new-layout

### PR DESCRIPTION
**Pull request by Sims24**
## Description

New layout for third-party integrations. 

Edit: 
Closes https://github.com/centreon/centreon-documentation/issues/973 https://github.com/centreon/centreon-documentation/pull/977 https://github.com/centreon/centreon-documentation/pull/976 https://github.com/centreon/centreon-documentation/pull/975 https://github.com/centreon/centreon-documentation/pull/974

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x (master)

### CODE REVIEW

File: fr/integrations/introduction-integrations.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Les intégrations reposent sur différents modules et fonctionnalités permettant de 
+s'adapter à vos processus de de traitement de données, d'alertes, d'événement et/ou 
+de métriques: 
+
+- [Centreon Stream Connector](https://github.com/centreon/centreon-stream-connector-scripts), 
+permet l'envoie de données d'événements et/ou de métriques vers un outil tiers afin 
+de partager, présenter ou retravailler les données dans un système annexe;
+- [Centreon Open Ticket](https://github.com/centreon/centreon-open-tickets), permet 
+l'ajout de fonctionnalité permettant l'ouverture manuelle de ticket sur la base des
```
Comment (tanguyvda) [inline fr/integrations/introduction-integrations.md:17]:
```suggestion
l'ajout de fonctionnalités permettant l'ouverture manuelle de ticket sur la base des 
```

File: fr/integrations/introduction-integrations.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [Centreon Stream Connector](https://github.com/centreon/centreon-stream-connector-scripts), 
+permet l'envoie de données d'événements et/ou de métriques vers un outil tiers afin 
+de partager, présenter ou retravailler les données dans un système annexe;
+- [Centreon Open Ticket](https://github.com/centreon/centreon-open-tickets), permet 
+l'ajout de fonctionnalité permettant l'ouverture manuelle de ticket sur la base des 
+données d'une alerte directement au travers des bacs à événements Centreon;
+- [Centreon Plugin Packs](../integrations/plugin-packs/introduction.html), permet la 
+récupération d'informations de supervision de systèmes tiers afin de permettre d'utiliser 
+Centreon comme la tour de contrôle permettant d'avoir un point unique de gestion 
+et de traitement des alertes sur vos équipements et applicatifs critiques.
```
Comment (tanguyvda) [inline fr/integrations/introduction-integrations.md:22]:
```suggestion
- [Centreon Plugin Packs](../integrations/plugin-packs/introduction.html), permet la 
récupération d'informations de supervision de systèmes tiers afin d'utiliser 
Centreon comme une tour de contrôle devenant le point unique de gestion 
et de traitement des alertes de vos équipements et applicatifs critiques.
```

Comment (tanguyvda) [reply fr/integrations/introduction-integrations.md:22]: 3 times "permet" doesn't feel good

File: fr/integrations/introduction-integrations.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+suivantes: 
+
+- *Data Analytics*: Envoyez vos données vers des solutions spécialisées dans le traitement,
+le stockage, et l'indexation de grands volumes de données polymorphes. Inclut également les 
+solutions permettant d'alerter sur la base de requêtes sur des outils de concentration de logs;
+- *Digital Experience Monitoring*: Récupérez des données de mesure de la performance
+et du ressenti utilisateur concernant vos sites et applicatifs directement depuis des 
+solutions avancées et dédiées au segment DEM;
+- *Event Management*: Utilisez les Stream Connector pour envoyer vos alertes vers 
+des outils spécialisées dans la gestion des événements en terme de qualification,
```
Comment (tanguyvda) [inline fr/integrations/introduction-integrations.md:35]:
```suggestion
des outils spécialisés dans la gestion des événements en terme de qualification, 
```

File: fr/integrations/introduction-integrations.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+le stockage, et l'indexation de grands volumes de données polymorphes. Inclut également les 
+solutions permettant d'alerter sur la base de requêtes sur des outils de concentration de logs;
+- *Digital Experience Monitoring*: Récupérez des données de mesure de la performance
+et du ressenti utilisateur concernant vos sites et applicatifs directement depuis des 
+solutions avancées et dédiées au segment DEM;
+- *Event Management*: Utilisez les Stream Connector pour envoyer vos alertes vers 
+des outils spécialisées dans la gestion des événements en terme de qualification, 
+d'escalades ou encore dans l'automatisation de leur traitement;
+- *ITSM*: Ouvrez des tickets depuis une ou plusieurs alertes détéctées par Centreon
+afin de permettre une gestion de l'incident et des actions corrective au plus proche
```
Comment (tanguyvda) [inline fr/integrations/introduction-integrations.md:38]:
```suggestion
afin de permettre une gestion de l'incident et des actions correctives au plus proche 
```

File: fr/integrations/introduction-integrations.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+et du ressenti utilisateur concernant vos sites et applicatifs directement depuis des 
+solutions avancées et dédiées au segment DEM;
+- *Event Management*: Utilisez les Stream Connector pour envoyer vos alertes vers 
+des outils spécialisées dans la gestion des événements en terme de qualification, 
+d'escalades ou encore dans l'automatisation de leur traitement;
+- *ITSM*: Ouvrez des tickets depuis une ou plusieurs alertes détéctées par Centreon
+afin de permettre une gestion de l'incident et des actions corrective au plus proche 
+de vos processus internes;
+- *Notifications*: Tirez partie des fonctionnalités de notifications du moteur Centreon Engine
+afin d'alerter de manière optimale les bonnes personnes au travers de mutliples modes
```
Comment (tanguyvda) [inline fr/integrations/introduction-integrations.md:41]:
```suggestion
afin d'alerter de manière optimale les bonnes personnes au travers de multiples modes 
```

File: fr/sidebars.json [hunk 6]
```diff
@@ -927,56 +927,65 @@
...
-                    "integrations/open-tickets/request-tracker-restapi",
-                    "integrations/open-tickets/serena",
-                    "integrations/open-tickets/servicenow"
+                    "integrations/event-management/sc-hp-bsm",
+                    "integrations/event-management/sc-hp-omi",
+                    "integrations/event-management/sc-opsgenie",
+                    "integrations/event-management/sc-pagerduty",
+                    "integrations/event-management/sc-servicenow",
+                    "integrations/event-management/sc-servicenow-mid-server"
                 ],
```
Comment (tanguyvda) [inline fr/sidebars.json:961]:
```suggestion
                    "integrations/event-management/sc-servicenow-mid-server",
                    "integrations/event-management/sc-kafka-events"
                ],
```

File: fr/sidebars.json [hunk 7]
```diff
@@ -927,56 +927,65 @@
...
-                    "integrations/open-tickets/mail",
-                    "integrations/open-tickets/otrs-restapi",
-                    "integrations/open-tickets/request-tracker-restapi",
-                    "integrations/open-tickets/serena",
-                    "integrations/open-tickets/servicenow"
+                    "integrations/event-management/sc-hp-bsm",
+                    "integrations/event-management/sc-hp-omi",
+                    "integrations/event-management/sc-opsgenie",
+                    "integrations/event-management/sc-pagerduty",
+                    "integrations/event-management/sc-servicenow",
```
Comment (tanguyvda) [inline fr/sidebars.json:959]: should be converted to sc-service-now-events as soon as the right file is in place (see previous review comment about replacing doc in french doc)

**Review by tanguyvda — CHANGES_REQUESTED** [review]

File: en/integrations/introduction-integrations.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Centreon provides numerous ways to cooperate with third-party tools to guarantee 
+a perfect integration within your IT ecosystem.
+
+Integrations rely on various modules and features to make Centreon compliant with 
+your data, alert, events, or metrics management processes: 
+
+- [Centreon Stream Connector](https://github.com/centreon/centreon-stream-connector-scripts), 
+get the capability to send event and metrics data to a third-party tool and sharing, 
+presenting, or extending the usage on top of your monitoring data.
```
[the remark this answers is not in the document]
Comment (ponchoh) [reply en/integrations/introduction-integrations.md:14]: Options for "on top of":

1. in addition to
2. including

[the remark this answers is not in the document]
Comment (Sims24) [reply en/integrations/introduction-integrations.md]: I rephrased it.

File: en/integrations/introduction-integrations.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+extension introduces additional UI actions and backend boiler-plates to create 
+cases within your favorite ticketing system. 
+
+- [Centreon Plugin Packs](../integrations/plugin-packs/introduction.html), offers 
+the possibility of grabbing data from other monitoring tools and making Centreon 
+the single pane of glass for all your alerts.
+
+We build this documentation on top of the features and value proposition third-party 
+solutions Centreon can connect to are offering. Here is a brief description of the 
+purpose of each of these sections:
```
[the remark this answers is not in the document]
Comment (Sims24) [reply en/integrations/introduction-integrations.md]: I rephrased it.

**Review by tanguyvda — APPROVED** [review]

**Pull request merged**